pykern/pkconst.py

```diff
@@ -5,6 +5,8 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from __future__ import absolute_import, division, print_function
+import math
+import scipy.constants
```

Comment:
pykern doesn't list scipy in the install_requires. On my mac I now see this
```python
pykern (135-rsbeams-rslaser-constants)$ python -c 'import pykern.pkdebug'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/ecarlin/src/radiasoft/pykern/pykern/pkdebug.py", line 71, in <module>
    from pykern import pkconfig
  File "/Users/ecarlin/src/radiasoft/pykern/pykern/pkconfig.py", line 75, in <module>
    from pykern import pkconst
  File "/Users/ecarlin/src/radiasoft/pykern/pykern/pkconst.py", line 9, in <module>
    import scipy.constants
ModuleNotFoundError: No module named 'scipy'
```

What about moving these consts to some other module (ex `pykern/pkmath.py`) and we are careful to not use that in context where scipy may not be installed?

Reply:
That's a good point. Do we want pykern to require scipy? I think so, but I could be convinced differently. That's the "big library" problem we had with BOP. scipy has the big library problem, too, but people don't seem to mind.

Reply:
> Here, I am contradicting the decision I made when originally writing rsconst.py but it now seems better that users should not go to RadiaSoft constants when they would just be duplicating something available in SciPy.

> For example, I suggest that m_e = scipy.constants.m_e and all analogous cases should be removed.

Will fix this.

Reply:
@gurhar1133 is correct.  The problem of dependence on scipy must still be addressed.

Reply:
I don't think we should but also could be convinced otherwise:). Here are some reasons:
- scipy [depends on py38](https://github.com/scipy/scipy/blob/b5d8bab88af61d61de09641243848df63380a67f/pyproject.toml#L58). We run py37. We want to upgrade but who knows when that will happen. I don't want to be forced to upgrade for a dep that provides as little to pykern as scipy does. But maybe being forced would be good...
- scipy [depends on numpy](https://github.com/scipy/scipy/blob/b5d8bab88af61d61de09641243848df63380a67f/pyproject.toml#L25). What version exactly seems confusing. Again, more dependency confusion.
- pykern is  an application support framework. If we have a dep on scipy (and then all the transitive deps that come with that) then we are depending on things that applications we support may depend on. Applications we are going to support will probably depend on scipy/numpy. Who knows what version. Seems like we are opening ourselves up to version conflicts.
- I think developing something where scientist can put pykern like things but more relevant for their domain makes more sense. Maybe a new package (ex rssci)? They could follow different coding standards, add whatever deps are relevant, etc.

Reply:
Another point, maybe they want to distribute on pypi which we don't currently do for pykern. If we were in different packages then that would be no problem.